### ISSUE TITLE
[1.16.5] Fatally missing registry entries, closing connection

### ISSUE THREAD

**Issue by GeorgeOrwell1** [TeamTwilight/twilightforest#1156]
Forge version: 
Twilight Forest version: 1.16.5-4.0.482

Link to crash log: [my latest.log](https://github.com/TeamTwilight/twilightforest/files/6549012/latest.log), [log of person who is trying to join LAN world](https://pastebin.com/7JD1VreR)

Steps to reproduce:
1. Install the latest version of Twilight Forest.
2. Go into a world, and open to LAN.
3. Ask someone (with the same version of Twilight Forest of course) to join your world.
4. Error message: "Fatally missing registry entries, closing connection"

What I expected to happen:
For the other person to be able to join my modded survival LAN world.

What happened instead:
Error message stating that there is a boss_spawner block that's missing (by looking in the logs) and therefore being unable to join.

**Comment by FloppaWithDrip** [comment]
Does it also do this if you make a new world? And are you absolutely 100% completely sure both of you are using the same version?

**Comment by GeorgeOrwell1** [comment]
> Does it also do this if you make a new world? And are you absolutely 100% completely sure both of you are using the same version?

It happened after upgrading to a newer version, and yes, we have the same version. However, it doesn't happen when creating a new world.

**Comment by Tamaized** [comment]
This sounds like a Forge bug then.

**Comment by GeorgeOrwell1** [comment]
> This sounds like a Forge bug then.

(How) can I resolve it?

**Comment by Tamaized** [comment]
You wait for this to be fixed https://github.com/MinecraftForge/MinecraftForge/issues/7737

**Comment by GeorgeOrwell1** [comment]
(I've tried it on a server, and can confirm that it's a Forge issue since it worked without problems by not using LAN.)

**Comment by Elrynon** [comment]
So my wife and I just had this issue happen to us. We updated the Val3 modpack and started getting the error. Since there hasn't been any "real" activity (just people saying they're having the same issue) in a month does that mean our world is now screwed until they decide to fix this bug? Is there anyway to roll back everything to what was working before we updated the modpack? I tried rolling back just the Val3 modpack but honestly that gave us more errors.
